HostSupport/CMakeLists.txt

```
@@ -17,6 +17,9 @@
 # along with Natron.  If not, see <http://www.gnu.org/licenses/gpl-2.0.html>
 # ***** END LICENSE BLOCK *****
 
+# see https://cmake.org/cmake/help/latest/module/FindEXPAT.html
+FIND_PACKAGE( EXPAT REQUIRED )
```

```suggestion
find_package(EXPAT REQUIRED)
```

I prefer CMake commands being lower case and whitespace trimmed for the arguments. Also this belongs to `CMakeLists.txt`.